Xamarin.ExposureNotification/ExposureNotification.android.cs

```diff
@@ -18,7 +23,7 @@ public static partial class ExposureNotification
 		static IExposureNotificationClient instance;
 
 		static IExposureNotificationClient Instance
-			=> instance ??= Nearby.GetExposureNotificationClient(Application.Context);
+			=> instance ??= Nearby.GetExposureNotificationClient(Xamarin.Essentials.Platform.CurrentActivity);
```

Comment:
What happens with multiple activities... should we cache?

The native sample uses the app context.

Reply:
If they use app we can just revert this. will do.